apis/v1alpha1/rhmi_types.go

```diff
@@ -86,7 +86,7 @@ var (
 	VersionAMQStreams          ProductVersion = "1.1.0"
 	VersionCodeReadyWorkspaces ProductVersion = "2.1.1"
 	VersionFuseOnOpenshift     ProductVersion = "7.6"
-	VersionMonitoring          ProductVersion = "1.7.0"
+	VersionMonitoring          ProductVersion = "1.8.0"
```

Comment:
There is also a `OperatorVersionMonitoring` a few lines down(line 116) variable that needs to be bumped too.